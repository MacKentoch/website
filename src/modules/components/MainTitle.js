import styled from 'styled-components'
import theme from 'style/theme'

const MainTitle = styled.h1`
  font-size: 40px;
  line-height: 65px;
  font-weight: 300;
  margin: 10px 0;

  @media (min-width: ${theme.medias.phablet}) {
    font-size: 50px;
  }
`

export default MainTitle
