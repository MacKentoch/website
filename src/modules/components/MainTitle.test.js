import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import MainTitle from './MainTitle'

describe('MainTitle', () => {
  it('should render a h1', () => {
    const wrapper = shallow(<MainTitle>Hello</MainTitle>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
