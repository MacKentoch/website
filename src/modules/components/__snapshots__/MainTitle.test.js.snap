// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainTitle should render a h1 1`] = `
<h1
  className="MainTitle-s1e8ozd8-0 iJrely"
>
  Hello
</h1>
`;
